src/packages/utils.ts

```diff
@@ -44,6 +44,16 @@ const sanitizePrivKey = (privKey: string): string => {
     return privKey;
 };
 
+const getRandomHexString = (): string => {
```

Comment:
We should be using at least 256 bit keys = 64 bytes in hex = 32 characters in hex.